inc/saiport.h

```diff
@@ -1501,6 +1501,14 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PRBS_POLYNOMIAL,
 
+    /**
+     * @brief Port PRBS receive error count
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_PRBS_RX_ERROR_COUNT,
```

Comment:
can this be in the port stat, rather than port attribute?

Reply:
move to port stats
https://github.com/opencomputeproject/SAI/pull/1179/commits/bed90dd32bff2db5c88a0821a7ac2f050c199b79